parts/k8s/kubernetesagentcustomdata.yml

```diff
@@ -166,14 +166,14 @@ coreos:
 {{else}}
 runcmd:
 - echo `date`,`hostname`, startruncmd>>/opt/m
-- retrycmd_if_failure() { for i in $(seq 1 36); do $@; [ $? -eq 0  ] && break || sleep 5; done; echo Executed \"$@\" $i times; }
```

Comment:
Is there anyway to not have this bash func repeated 3 times and centralize it somewhere? It's in master custom data twice and once in agent custom data. Any suggestions?